cornice/schemas.py

```diff
@@ -175,3 +175,78 @@ def _validate_fields(location, data):
         for param in set(params) - set([attr.name for attr in attrs]):
             request.errors.add('body' if param in body else 'querystring',
                                param, msg % param)
+
+
+def validate_matchdict(matchdict_types):
```

Comment:
That's an interesting approach but I don't know if we want to have that in core cornice though. @leplatrem @tarekziade thoughts?